README.rst

```diff
@@ -266,10 +266,25 @@ of the endpoint parameter ``message`` to your view function.
 
 .. note:: In the OpenAPI 3.x.x spec, the requestBody does not have a name.
           By default it will be passed in as 'body'. You can optionally
-          provide the x-body-name parameter in your requestBody schema
+          provide the x-body-name parameter in your operation 
+          (or legacy position within the requestBody schema)
```

Comment:
There was still one I missed 😅 Although not sure what the clearest explanation is here with `requestBody` and `requestBody schema`
```suggestion
          provide the x-body-name parameter in your requestBody
          (or legacy position within the requestBody schema)
```

Reply:
I think this is fine :slightly_frowning_face: